tests/TestCase/Routing/RouteBuilderTest.php

```diff
@@ -494,7 +494,7 @@ public function testNestedPrefix()
         $routes = new RouteBuilder($this->collection, '/admin', ['prefix' => 'admin']);
         $res = $routes->prefix('api', ['_namePrefix' => 'api:'], function ($r) {
             $this->assertSame('/admin/api', $r->path());
-            $this->assertEquals(['prefix' => 'admin/api'], $r->params());
+            $this->assertEquals(['prefix' => 'admin/Api'], $r->params());
```

Comment:
Shouldn't it be `Admin` too instead of `admin`?

Reply:
No because the 'admin' value was passed into the RouteBuilder as a lower case value. This likely won't happen often in applications because people use the scope() helpers.

Reply:
So `'_namePrefix' => 'contacts:']` also won't change then?

Reply:
`_namePrefix` is used for building `_name`, it is not related to `prefix`.